hapi/hedera-protobufs/services/state/blockstream/block_stream_info.proto

```diff
@@ -48,36 +53,36 @@ option java_multiple_files = true;
 message BlockStreamInfo {
     /**
      * A block number.<br/>
-     * This is the block number of the last completed block.
+     * This is the current block number.
      */
-    uint64 last_block_number = 1;
+    uint64 block_number = 1;
 
     /**
-     * A block hash value.<br/>
-     * This is the hash of the last completed block.
-     */
-    bytes last_block_hash = 2;
-
-    /**
-     * A consensus time for the last completed block.
-     * This is used to determine if this block was the first across an
+     * A consensus time for the current block.<br/>
+     * This is the _first_ consensus time in the current block, and
```

Comment:
It's a bit unclear what "the first consensus time in the current block" actually means. Is it the consensus time of the first (earliest?) event in this block, and there's some other events with later (newer) consensus times in this block? Or is it the time when the network reached a consensus on this block for the first time? Or...?

I suggest to expand this doc to clarify what this timestamp means exactly.

Reply:
Clarified as the round timestamp of the first round in the block.

Reply:
Thanks!